frontend/lib/requests/droplet.ts

```
@@ -495,6 +518,683 @@ export async function createDroplet(data: z.infer<typeof CreateDropletSchema>) {
+          console.log(`Deleted lesson ${lesson.id}`);
+        } catch (error) {
+          console.error(`Error deleting lesson ${lesson.id}:`, error);
+          // Continue with other lessons even if one fails
+        }
+      }
+    }
+
+    // Update the original droplet with draft data
+    const updatedName = draftDroplet.name.replace(/^\[EDIT\]\s*/i, "");
+
+    const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)